frontend/src/views/ManageUsersView.vue

```diff
@@ -67,27 +65,21 @@
                         <template v-slot:bottom><!-- no paging --></template>
                       </v-data-table>
                     </v-col>
-                    <v-col cols="6">
-                    </v-col>
+                    <v-col cols="6"></v-col>
                   </v-row>
                 </td>
                 <td class="pl-0">
-                  <AppButton variant="text">Deactivate user</AppButton>
+                  <AppButton v-if="showDeactivateUserButton(item)" variant="text" @click="toggleDeactivateUserDialog(item)">Deactivate user</AppButton>
                 </td>
               </tr>
             </template>
           </v-data-table>
         </v-skeleton-loader>
       </v-col>
     </v-row>
-    <ManageUserDialog
-      class="pa-0"
-      :show="showManageUserDialog"
-      :updatingUser="userToUpdate"
-      @close="toggleDialog"
-      @close-refresh="closeDialogAndRefresh"
-      @update-success-event="updateSuccessEvent" />
-    <AppButton id="back-account-mgn-button" class="mt-2" :primary="false" size="large" width="400px" :to="{ name: 'account-mgmt' }">&larr; Back to Account Management</AppButton>
+    <ManageUserDialog class="pa-0" :show="showManageUserDialog" :updatingUser="userToUpdate" @close="toggleDialog" @close-refresh="closeDialogAndRefresh" @update-success-event="updateSuccessEvent" />
```

Comment:
Do we need this class? If so can it be moved into ManageUserDialog.vue?

Reply:
Removed, not necessary.